Source/WebCore/css/parser/CSSParserImpl.cpp

```diff
@@ -322,7 +323,16 @@ void CSSParserImpl::parseStyleSheetForInspector(const String& string, const CSSP
 
 static CSSParserImpl::AllowedRules computeNewAllowedRules(CSSParserImpl::AllowedRules allowedRules, StyleRuleBase* rule)
 {
-    if (!rule || allowedRules == CSSParserImpl::AllowedRules::FontFeatureValuesRules || allowedRules == CSSParserImpl::AllowedRules::KeyframeRules || allowedRules == CSSParserImpl::AllowedRules::CounterStyleRules || allowedRules == CSSParserImpl::AllowedRules::ViewTransitionRules || allowedRules == CSSParserImpl::AllowedRules::NoRules)
+    // FIXME: name suggestions?
+    bool ruleIsUnrestricted = !rule
```

Comment:
ruleUsesAllowedRules ?

Reply:
A general principle is to name booleans starting with a yes/no verb, like `isUnrestrictedRule`.

A switch might be better here.

Reply:
Unrestricted is probably not the right verb fwiw, `allowedRules` are a set of restrictions too. Maybe just drop the variable and use a switch as Antti says.

Reply:
We dropped some `AllowRules` enums so the `if` condition is shorter now, hence I'll just keep the `if`. But yeah, I might make it into a switch if I have time.